cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -759,3 +771,39 @@ def test_standard_gates_wrong_params_error(qasm_gate: str, num_params: int):
                        match=r".*{}.* takes {}.*got.*0.*line 4".format(
                            qasm_gate, num_params)):
         parser.parse(qasm)
+
+
+@pytest.mark.parametrize('qasm_gate,cirq_gate', single_qubit_gates)
+def test_single_qubit_gates(qasm_gate: str, cirq_gate: cirq.SingleQubitGate):
+    qasm = """
+     OPENQASM 2.0;
+     include "qelib1.inc";
+     qreg q[2];
+     {0} q[0];
+     {0} q;
+     {0}() q;
```

Comment:
This is valid syntax?

Reply:
Yes! 
![Screenshot_20190712-205858](https://user-images.githubusercontent.com/635073/61164922-008b2e00-a4e8-11e9-90e2-c0bc4d504298.png)

Reply:
On the other hand Quantum experience doesn't like it
![Screenshot_20190712-211929](https://user-images.githubusercontent.com/635073/61165177-e3a42a00-a4ea-11e9-8754-5dcc74b4b230.png)